maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -299,6 +339,67 @@ class Orchestra(
         return false
     }
 
+    private fun assertNoDefectsWithAICommand(command: AssertNoDefectsWithAICommand): Boolean = runBlocking {
+        // TODO(bartekpacia): make all of Orchestra suspending
+
+        if (ai == null) {
+            throw MaestroException.AINotAvailable("AI client is not available. Did you export $AI_KEY_ENV_VAR?")
+        }
+
+        val imageData = Buffer()
+        maestro.takeScreenshot(imageData, compressed = false)
+
+        val defects = Prediction.findDefects(
+            aiClient = ai,
+            screen = imageData.copy().readByteArray(),
+            previousFalsePositives = listOf(), // TODO(bartekpacia): take it from WorkspaceConfig (or MaestroConfig?)
```

Comment:
Why do we have this right now? I think it would be useful to use it whenever we know the user feedback.

Reply:
Yeah, I agree and this was my intention. I was just thinking out loud and that's why I wrote down that TODO>

Overall, I don't mean the `TODO`s as having to be resolved immediately in this PR. Perhaps we should set on a single standard of how we interpret `TODO`s, `FIXME`s, etc :)

Reply:
Yeah sounds good! From interpretation, TODOs are small inclusions that can be made as quick followups which in this case I don't think is true since this highly depends on user feedback.

Reply:
Gotcha. I removed this code.